stories/quick-view/quick-view.stories.js

```diff
@@ -0,0 +1,258 @@
+                    </div>
+                </div>
+                
+                <div class="fd-form-group" role="group">
+                    <div class="fd-form-item">
+                        <label class="fd-form-label">Start Date:</label>
+                        <span class="fd-input">01/01/2015</span>
+                    </div>
+                    <div class="fd-form-item">
+                        <label class="fd-form-label">End Date:</label>
+                        <span class="fd-input">31/12/2015</span>
+                    </div>
+                    <div class="fd-form-item">
+                        <label class="fd-form-label">Occurrence:</label>
+                        <span class="fd-input">Weekly</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+NoHeader.storyName = 'Without the header';
+
+NoHeader.parameters = {
+    docs: {
+        iframeHeight: 500,
+        storyDescription: 'QuickView without the header'
+    }
+};
+
+export const RTLMode = () => `<div class="fd-popover">
+    <div class="fd-popover__body" aria-hidden="false" id="popoverA1">
+        <div class="fd-quick-view" dir="rtl">
+            <div class="fd-bar fd-bar--header">
+                <div class="fd-bar__middle">
+                    <div class="fd-bar__element">Company</div>
+                </div>
+            </div>
+
+            <div class="fd-quick-view__content">
+                <div class="fd-bar fd-bar--header-with-subheader">
+                    <div class="fd-bar__left">
+                        <span class="fd-avatar fd-avatar--s" role="presentation">
+                            <i role="presentation" class="fd-avatar__icon sap-icon--camera"></i>
+                        </span>
+    
+                        <div class="fd-quick-view__subheader-text">
+                            <h5 class="fd-title fd-title--h5">
+                                <a class="fd-link" href="#">Inventarisation</a>
+                            </h5>
+                        </div>
+                    </div>
+                </div>
+                
+                <div class="fd-form-group" role="group">
+                    <div class="fd-form-item">
+                        <label class="fd-form-label">Start Date:</label>
+                        <span class="fd-input">01/01/2015</span>
+                    </div>
+                    <div class="fd-form-item">
+                        <label class="fd-form-label">End Date:</label>
+                        <span class="fd-input">31/12/2015</span>
+                    </div>
+                    <div class="fd-form-item">
+                        <label class="fd-form-label">Occurrence:</label>
+                        <span class="fd-input">Weekly</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+RTLMode.storyName = 'RTL mode';
```

Comment:
Let's also remove RTL mode example, there is a way to control direction on docs header
![image](https://user-images.githubusercontent.com/26483208/102872199-b4178200-443f-11eb-8cf3-f16d653a87e8.png)

Reply:
changed